Sources/xcodeproj/XCScheme+BuildableProductRunnable.swift

```diff
@@ -0,0 +1,42 @@
+import AEXML
```

Comment:
You could also potentially put these files in an `XCScheme` group and drop the `XCScheme+` from the filename. It would clean up the list of files a bit as there are so many now

Reply:
👍